jekyll/assets/css/footer.css

```diff
@@ -2,6 +2,8 @@
  * Footer
  * */
 .footer {
+  position: relative;
```

Comment:
Are there other styles that are making us require this? I tried removing it and didn't see many changes 😄 From what I read on [css tricks](https://css-tricks.com/almanac/properties/p/position/), I _think_ `relative` is the same as the default `static`, except it lets you do positioning with `top`/`left` etc.

In my experience, I've only used `position: relative` when I'm also messing around with `position: absolute`

Reply:
The reason I did this was actually for z-indexing reasons. If the element is positioned relative, it responds to a z-index position, which initial positioning (static) does not allow for. 

I needed it to respond to z-index position so that it sits on top of the sidenav (necessary to prevent ugliness on scrolling).